src/go/MONIT/annotationManager.go

```diff
@@ -117,6 +120,8 @@ func getAnnotations(data *[]annotationData, tags []string) {
 		v.Add("tags", strings.Trim(tag, " "))
 	}
 
+	v.Add("limit", strings.Trim(grafanaLimit, " "))
```

Comment:
there is no need for Trim once you convert grafanaLimit to int, just use fmt.Sprintf("%d", grafanaLimit)

Reply:
fixed.